spec/factories/contacts.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
```

Comment:
You can use https://github.com/stympy/faker to generate fake 'random' test data.

Reply:
Yeah I would do this when I start writing tests